File: push/src/main/java/com/xiaomi/push/service/awake/module/HelpType.java
package com.xiaomi.push.service.awake.module;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/push/service/awake/module/HelpType.class */
public enum HelpType {
    ACTIVITY("activity"),
    SERVICE_ACTION("service_action"),
    SERVICE_COMPONENT("service_component"),
    PROVIDER("provider");

    public String typeValue;

    HelpType(String str) {
        this.typeValue = str;
    }
}

File: legacy/src/main/java/com/xiaomi/push/service/awake/module/HelpType.kt
package com.xiaomi.push.service.awake.module
import io.github.magisk317.mipush.protocol.model.*

enum class HelpType(
    @JvmField val typeValue: String,
) {
    ACTIVITY("activity"),
    SERVICE_ACTION("service_action"),
    SERVICE_COMPONENT("service_component"),
    PROVIDER("provider"),
}
